src/services/completions.ts

```diff
@@ -983,6 +985,7 @@ namespace ts.Completions {
                             symbol.declarations.some(d => d.kind !== SyntaxKind.SourceFile && d.kind !== SyntaxKind.ModuleDeclaration && d.kind !== SyntaxKind.EnumDeclaration)) {
                             addTypeProperties(typeChecker.getTypeOfSymbolAtLocation(symbol, node));
                         }
+                        setSortTextToOptionalMember();

```

Comment:
I don't think issue you are fixing asks for sorting of member completions based on optional member? checking with @DanielRosenwasser for that.

```ts
interface I {
a?: string;
b: number;
}

const x: I;
x./** here should a be shown after b */
```

Reply:
Not 100% sure if this is what you're asking, but the feature was definitely that optional properties should come after required properties.

Reply:
The issue referenced contains the scenario: 
```ts
interface I { a?: string; b: number; }
const x: I = {  /*completion here should have a coming after b*/ }
```

I am asking if this is also intended scenario
```ts
interface I { a?: string; b: number; }
const x: I;
x./*completion here*/
``

Reply:
Since I think they are all about properties completion, so I made this change, as well as properties inside class and JSX element. I can fix that if you think they are not related to the issue. @DanielRosenwasser @sheetalkamat

Reply:
Ah, no, I meant to only prioritize completions for object literals properties and JSX attributes. Sorry for any ambiguity, and thank you @sheetalkamat for catching that.